src/stdlib_utils/constants.py

```diff
@@ -9,6 +9,8 @@
 from typing import Union
 
 SECONDS_TO_SLEEP_BETWEEN_CHECKING_QUEUE_SIZE = 0.05
+QUEUE_CHECK_TIMEOUT_SECONDS = 0.2
+QUEUE_DRAIN_TIMEOUT_SECONDS = 0.02
```

Comment:
why is there a different timeout for checking vs draining? Please add some documentation about why there are 2 different timeouts.

I thought that in general it sometimes took up to `0.2` seconds in our experience in GitHub CI for a queue with an object in it to actually return something when `.get` was called.  I know in the desktop_app we have something separate for called `SECONDS_TO_WAIT_WHEN_POLLING_QUEUES`, but I'm not actually 100% sure if that's the gating factor at the moment for polling inside the subprocesses....I think(?) in most cases in the tests we just switched to confirming that an object was placed into a queue before invoking the subprocess to run(?).  (and when the subprocess in actually running in an infinite loop, then the timeout in the `.get` doesn't really matter---I don't think---because it's just infinite looping)

Reply:
I can't remember the exact reason, but I would think it's because we want to drain as fast as possible when hard-stopping

Reply:
if the queue isn't actually drained though, then the process will still hang when we attempt to hard stop it though...won't it?  The goal of draining the queues is to ensure they're empty before attempting to join the process, right?